config/environments/production.rb

```diff
@@ -22,28 +20,25 @@
 
   # Disable serving static files from the `/public` folder by default since
   # Apache or NGINX already handles this.
-  config.public_file_server.enabled = ENV["RAILS_SERVE_STATIC_FILES"].present?
+  config.public_file_server.enabled = ENV['RAILS_SERVE_STATIC_FILES'].present?
 
-  # Compress JavaScripts and CSS.
-  config.assets.js_compressor = Uglifier.new(harmony: true)
```

Comment:
Why did this change?

Reply:
This is another `app:update` template change. I'm not sure if this needs to go away or not. I think it goes away for sure if we switch to webpacker.